packages/frontend/core/src/components/affine/setting-modal/general-setting/editor/edgeless/docs/index.ts

```diff
@@ -28,18 +28,27 @@ export type DocName =
 const docMap = new Map<DocName, Promise<Doc | undefined>>();
 
 export async function getDocByName(name: DocName) {
-  const rawData: Record<DocName, any> = {
-    note: (await import('./note.json')).default,
-    pen: (await import('./pen.json')).default,
-    shape: (await import('./shape.json')).default,
-    flow: (await import('./flow.json')).default,
-    text: (await import('./text.json')).default,
-    connector: (await import('./connector.json')).default,
-    mindmap: (await import('./mindmap.json')).default,
-  };
   if (docMap.get(name)) {
     return docMap.get(name);
   }
+  const [note, pen, shape, flow, text, connector, mindmap] = await Promise.all([
```

Comment:
I think we can
1. load snapshot on requesting specific doc snapshot
2. cache import (may be unnecessary)

Reply:
Code updated. Please take a look.